.prettierignore

```diff
@@ -21,6 +21,8 @@ build/
 /files/fr/glossary/main_axis/index.md
 /files/fr/learn/server-side/django/forms/index.md
 /files/pt-br/learn/server-side/django/forms/index.md
+/files/ru/learn/server-side/django/forms/index.md
+/files/ru/learn/server-side/django/introduction/index.md
```

Comment:
@queengooborg the same question [as in `/learn/html` PR](https://github.com/mdn/translated-content/pull/14539#pullrequestreview-1546958130).

Reply:
No relevant anymore, thanks